mvn/cloudbuild.yaml

```diff
@@ -60,6 +60,9 @@ steps:
 - name: 'gcr.io/$PROJECT_ID/mvn:gcloud'
   args: ['install']
   dir: 'examples/spring_boot'
+- name: 'gcr.io/$PROJECT_ID/mvn:appengine'
+  args: ['package', 'appengine:stage']
```

Comment:
this runs a command that doesn't call out to a remote service but still requires the cloud sdk